app/controllers/lookbook/previews_controller.rb

```diff
@@ -20,10 +20,15 @@ def index
               {
                 name: preview.name,
```

Comment:
```suggestion
                name: preview.name,
                lookup_path: preview.lookup_path,
```

Wondering if we could add this small other thing too? Would end up adding `primer/alpha/component`